lib/betagouvbot/mail_action.rb

```diff
@@ -9,11 +9,11 @@ def initialize(client, mail)
     end
 
     def subject
-      @mail[:personalizations][0][:subject]
+      @mail[:personalisations][0][:subject]
```

Comment:
I'm concerned SendGrid might not do the right thing with this alternative spelling. How did you test  this?